bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/colorlogic.xml

```diff
@@ -30,8 +30,8 @@
 		<description>Light State</description>
 		<state readOnly="true">
 			<options>
-				<option value="0">Off</option>
-				<option value="1">On</option>
+				<option value="0">OFF</option>
```

Comment:
As label, "On" and "Off" are better than all in capitals. After all, option 7 is also called "Powering Off". So please revert this change.